konflux-ci/argocd/argocd-namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
This should not be deployed when using the EKS cluster and Terraform take care of deploying ArgoCD.